lib/wibox/widget/slider.lua

```diff
@@ -505,6 +507,45 @@ function slider:fit(_, width, height)
     return width, height
 end
 
+function slider:layout(context, width, height)
```

Comment:
Maybe add `if not self._private.handle_widget then return {} end`? That depends if you add / plan_to_add a bar widget too.